docs/arm.rst

```diff
@@ -107,7 +107,7 @@ TBB, Parallel STL, BLAS, LAPACK) may cause compatibility issues if they are not
 the same version as the one used by Open3D.
 
 If you only need the Python wheel, consider using the Docker build method or
-install Open3D via ``pip install open3d`` directly.
+install Open3D via ``pip install open3d[standard]`` directly.
```

Comment:
Same, try to provide something like "if you only need the core functionalities and do not need visualization, you may install the minimal wheel via `pip install open3d`".